src/components/LastAction.tsx

```diff
@@ -0,0 +1,46 @@
+import type { FC } from "react";
+import { useSelector } from "react-redux";
+import styled from "styled-components";
+
+import { actions as hijackedActions } from "../redux/actions";
```

Comment:
hijackedActions - ten název sis vymyslel nebo jsi to někde našel? 😀

Reply:
Oh ano naming... :D Chcel som zobrazovať effekty ktoré v appke prebiehajú, nevedel som však získať akcie priamo z RTK Query tak som ich na tvrdo "hijack"-ol. Nie je to úplne pekné, no ako-tak to funguje.